### ISSUE TITLE
A word of thanks!

### ISSUE THREAD

**Issue by dmoul** [axismaps/colorbrewer#38]
Just a quick note to say that I use this a lot and find myself recommending it again and again (and did so again today). 

Thanks so much for making this available and keeping it alive.

--Daniel Moul

**Comment by gitressa** [comment]
I agree, this is an incredible tool. The only thing I can think of which would improve it, would be more color schemes.

Or even add an option of defining custom start, middle, and destination color?

**Comment by davidheyman** [comment]
Thanks for the kind words! Closing this as it's a message rather than an actionable issue.
